src/ui/counter.ts

```diff
@@ -49,9 +53,11 @@ export class Counter3D {
     this.maxDigits = maxDigits
     this.zeroPadding = zeroPadding
     this.id = id
+    this.justify = justify ? justify : 'center'
```

Comment:
```suggestion
    this.justify = justify ?? 'center'
```

Reply:
you can start using the nullish operator if you want.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing